src/main/java/software/amazon/msk/auth/iam/internals/AuthenticationRequestParams.java

```diff
@@ -60,11 +62,11 @@ public String getServiceScope() {
     public static AuthenticationRequestParams create(@NonNull String host,
             AwsCredentials credentials,
             @NonNull String userAgent) throws IllegalArgumentException {
-        Region region = Optional.ofNullable(regionMetadata.tryGetRegionByEndpointDnsSuffix(host))
+        com.amazonaws.regions.Region region = Optional.ofNullable(regionMetadata.tryGetRegionByEndpointDnsSuffix(host))
```

Comment:
Nit: Any reason this is fully qualified?

Reply:
Yes, until full transition to java-sdk-v2, I added a class to transform V1 Region to/from V2 Region. In AuthenticationRequestParams I still need to work with both, so by default, it is the V2 Region that is imported (at L30) and stored (at L54), but I still need to retrieve the configured region from a class that return a V1 Region, the case in this line (L65). And you can see that at the end, the V1 region is transformed in V2 region (L70).

It should be removed in https://github.com/aws/aws-msk-iam-auth/pull/164